config/locales/en/main.yml

```diff
@@ -650,9 +651,10 @@ en:
       questioning:
         default: "Text entered here will be pre-filled in the answer space (for ODK only). You can enter $QuestionCode to include the value of a previous question, and $!RepeatNum to include the number of the current item in a repeat group.\n\nFor example, entering\n\n&nbsp;&nbsp;&nbsp;ID: $Household-$!RepeatNum\n\nwould pre-fill the answer with\n\n&nbsp;&nbsp;&nbsp;ID: 176-2\n\nfor the second person in household 176, assuming you have a question with code 'Household'. You can also enter an XPath calculate expression by wrapping it with <code>calc()</code>:\n\n&nbsp;&nbsp;&nbsp;<code>calc($VillageNum + 100 / 2)</code>"
         display_logic: "Conditions which must be met in order for this question to be shown to the user."
-        hidden: "Whether this question will be shown on this form. Use this to disable a question if you can't delete it. This value will override the condition and required settings below."
-        read_only: "If checked, the enumerator won't be able to change the pre-filled value when using ODK. Does not affect web or SMS responses."
-        required: "Whether this question must be completed on this form or is optional. Only enforced if question is not hidden the condition specified above is met."
+        hidden: "Hide question but still collect default answer (ODK only)."
```

Comment:
More specific question about wording: `Hide question but still collect default answer (ODK only)` makes me assume the checkbox has **no effect** outside of ODK, when in fact it causes the question to behave as if it's **disabled** for web submissions.

Like you said, maybe we can tweak wording when we demo the checkbox sandwich.